product_docs/docs/biganimal/release/administering_cluster/01_portal_access.mdx

```diff
@@ -58,7 +58,7 @@ The following roles grant privileges within a project:
   - View and download a usage report
 
   !!!note
```

Comment:
Does there need to be a space between '!!!' and 'note' here?

Reply:
https://github.com/EnterpriseDB/docs/blob/develop/README.md#admonitions-notes-warnings-etc